src/web/server/htmlTemplate.ts

```diff
@@ -103,6 +103,34 @@ export const htmlTemplate = ({
 
     const twitterMetaTags = generateMetaTags(twitterData, 'name');
 
+    // Duplicated prefetch and preconnect tags from DCP:
+    const staticPrefetchUrls = [
+        `https://assets.guim.co.uk/`,
+        `https://i.guim.co.uk`,
+        `https://api.nextgen.guardianapps.co.uk`,
+        `https://hits-secure.theguardian.com`,
+        `//j.ophan.co.uk`,
+        `//ophan.theguardian.com`,
+        `//phar.gu-web.net`,
+        `//www.google-analytics.com`,
+        '//sb.scorecardresearch.com',
```

Comment:
Can we add a comment around why these are commented out? Or maybe just remove them?

Reply:
they're not commented out - `//` means use `https://` if it's available or `http://` otherwise. I think probably everything is https at this point, and this is just legacy from when it was implemented on DCP. I will check that https works for all of those and update accordingly.

Reply:
they arent commented out, but they just do not have the `https:` defined. I would be curious as to why? are these http and not https requests?

Reply:
They probably were at some point but i've updated